rust/processor/src/schema.rs

```diff
@@ -518,6 +539,80 @@ diesel::table! {
     }
 }
 
+diesel::table! {
+    fungible_asset_activities (transaction_version, event_index) {
+        transaction_version -> Int8,
+        event_index -> Int8,
+        #[max_length = 66]
+        owner_address -> Varchar,
+        #[max_length = 66]
+        storage_id -> Varchar,
+        #[max_length = 1000]
+        asset_type -> Varchar,
+        is_frozen -> Nullable<Bool>,
+        amount -> Nullable<Numeric>,
+        #[sql_name = "type"]
+        type_ -> Varchar,
+        is_gas_fee -> Bool,
+        #[max_length = 66]
+        gas_fee_payer_address -> Nullable<Varchar>,
+        is_transaction_success -> Bool,
+        #[max_length = 1000]
+        entry_function_id_str -> Nullable<Varchar>,
```

Comment:
Is this going to empty if this activity is from a script tx? Also, in the case of multisig tx execution, what is this set to?

Reply:
Payload is not going to be an entry function payload (it's a multisig execution payload that wraps around entry or script payload)

Reply:
Hmm. Do you have an example of this? We can check how it shows up currently.

Reply:
I don't have a tx that changes some coin balances but here's a multisig tx: https://explorer.aptoslabs.com/txn/207817466/payload?network=mainnet

Reply:
Got it, transaction_payload shows up as null so we will show null as well b/c we get it from the API.

Reply:
Okay. That's fine for now